frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -286,17 +285,19 @@ function BookInfo:onShowBookCover(file, force_orig)
     end
 end
 
-function BookInfo:getCoverPageImage(doc, file, force_orig)
+function BookInfo:getCoverPageImage(doc, file, no_custom_cover)
     local cover_bb
-    local custom_cover = DocSettings:getCustomBookCover(file or (doc and doc.file))
-    if not force_orig and custom_cover then
-        local ImageWidget = require("ui/widget/imagewidget")
-        local img_widget = ImageWidget:new{
-            file = custom_cover,
-            file_do_cache = false,
-        }
-        cover_bb = img_widget:getImageCopy()
-        img_widget:free()
+    if not no_custom_cover then
+        local custom_cover = DocSettings:getCustomBookCover(file or (doc and doc.file))
+        if custom_cover then
```

Comment:
Feels like this won't handle the common case of `no_custom_cover = nil/false` and no custom cover has been set: it won't go in the `else` to fetch the original cover from the book ?

Reply:
So returning to my original code.